frontend/src/components/ContextMenu.js

```diff
@@ -31,8 +31,14 @@ function ContextMenu({
       menuWidthOffset={menuWidthOffset}
       up={up}
       menuHeightOffset={menuHeightOffset}
-      buttonText={<FontAwesomeIcon color={colors.textInk} icon={faEllipsisH} />}
-      buttonTestId="ellipsis-button"
+      buttonText={(
+        <>
+          <span className="usa-button--unstyled">Actions</span>
+          <span>&nbsp;</span>
+          <FontAwesomeIcon color={colors.textLink} icon={faChevronDown} />
+        </>
+)}
+      buttonTestId="actions-button"
```

Comment:
My only suggestion would be to consider having this data test ID to be even more unique named than `actions-button` to ensure there is never a conflict with another test ID and to give more context to what the button represents if encountered without these contextual details. Something like `buttonTestId=context-menu-actions-btn`.